### PR TITLE
Update adding 2.62 items

### PR DESCRIPTION
**Pull request by Josemanki**
Updates to 2.62 items
White rat and Black rat sets were overwritten and not remade at the bottom
Everything should conform to the format correctly.

### CODE REVIEW

File: server/app/database/data/sets.json [hunk 1]
```diff
@@ -15691,128 +15691,118 @@
     "bonuses": {
       "2": [
         {
-          "stat": "Vitality",
-          "value": 10,
+          "stat": "Power",
+          "value": 40,
```
Comment (samlee405) [inline server/app/database/data/sets.json:15695]: 20

File: server/app/database/data/sets.json [hunk 2]
```diff
@@ -15691,128 +15691,118 @@
...
         {
-          "stat": "Vitality",
-          "value": 15,
-          "altStat": null
-        },
-        {
-          "stat": "Agility",
-          "value": 10,
+          "stat": "Power",
+          "value": 40,
```
Comment (samlee405) [inline server/app/database/data/sets.json:15702]: 30

File: server/app/database/data/sets.json [hunk 3]
```diff
@@ -15691,128 +15691,118 @@
...
           "altStat": null
         },
         {
-          "stat": "Strength",
-          "value": 10,
+          "stat": "AP Reduction",
+          "value": 7,
```
Comment (samlee405) [inline server/app/database/data/sets.json:15707]: 3

File: server/app/database/data/sets.json [hunk 4]
```diff
@@ -15691,128 +15691,118 @@
...
-          "value": 15,
+          "stat": "Dodge",
+          "value": 5,
           "altStat": null
         },
         {
-          "stat": "Strength",
-          "value": 15,
+          "stat": "AP Reduction",
+          "value": 7,
```
Comment (samlee405) [inline server/app/database/data/sets.json:15724]: 5

File: server/app/database/data/sets.json [hunk 5]
```diff
@@ -15971,148 +15961,118 @@
...
-        {
-          "stat": "Vitality",
-          "value": 10,
-          "altStat": null
-        },
-        {
-          "stat": "Trap Damage",
-          "value": 1,
+          "stat": "Power",
+          "value": 40,
```
Comment (samlee405) [inline server/app/database/data/sets.json:15965]: 20 here

File: server/app/database/data/sets.json [hunk 6]
```diff
@@ -15971,148 +15961,118 @@
...
-        {
-          "stat": "Vitality",
-          "value": 20,
-          "altStat": null
-        },
-        {
-          "stat": "Power (traps)",
-          "value": 5,
+          "stat": "Power",
+          "value": 40,
```
Comment (samlee405) [inline server/app/database/data/sets.json:15972]: 30

File: server/app/database/data/sets.json [hunk 7]
```diff
@@ -15971,148 +15961,118 @@
...
           "altStat": null
         },
         {
-          "stat": "Trap Damage",
-          "value": 2,
+          "stat": "AP Parry",
+          "value": 7,
```
Comment (samlee405) [inline server/app/database/data/sets.json:15977]: 3

File: server/app/database/data/sets.json [hunk 8]
```diff
@@ -15971,148 +15961,118 @@
...
-          "value": 10,
+          "stat": "Lock",
+          "value": 5,
           "altStat": null
         },
         {
-          "stat": "Trap Damage",
-          "value": 4,
+          "stat": "AP Parry",
+          "value": 7,
```
Comment (samlee405) [inline server/app/database/data/sets.json:15994]: 5

File: server/app/database/data/weapons.json [hunk 9]
```diff
@@ -23998,67 +23998,89 @@
...
+        "minStat": 3,
+        "maxStat": 5
       }
     ],
     "weaponStats": {
-      "apCost": 5,
+      "apCost": 4,
       "usesPerTurn": 1,
-      "minRange": null,
+      "minRange": 1,
```
Comment (samlee405) [inline server/app/database/data/weapons.json:24059]: null

File: server/app/database/data/weapons.json [hunk 10]
```diff
@@ -23998,67 +23998,89 @@
...
         {
-          "stat": "Neutral steal",
-          "minStat": 3,
-          "maxStat": 5
+          "stat": "Water Damage",
+          "minStat": 16,
+          "maxStat": 28
+        },
+        {
+          "stat": "AP (target)",
```
Comment (samlee405) [inline server/app/database/data/weapons.json:24075]: Just `Ap` for stat

File: server/app/database/data/weapons.json [hunk 11]
```diff
@@ -24149,40 +24171,65 @@
...
+        "minStat": 3,
+        "maxStat": 5
+      },
+      {
+        "stat": "MP Parry",
+        "minStat": 3,
+        "maxStat": 5
       }
     ],
     "weaponStats": {
```
Comment (samlee405) [inline server/app/database/data/weapons.json:24235]: weapon damage has been modified for the daggers. 13-22 neutral and 4-6 air steal

File: server/app/database/data/weapons.json [hunk 12]
```diff
@@ -59100,5 +59129,470 @@
...
+      {
+        "stat": "% Air Resistance",
+        "minStat": 4,
+        "maxStat": 5
+      }
+    ],
+    "weaponStats": {
+      "apCost": 5,
+      "usesPerTurn": 1,
+      "minRange": 1,
```
Comment (samlee405) [inline server/app/database/data/weapons.json:59215]: null

File: server/app/database/data/weapons.json [hunk 13]
```diff
@@ -59100,5 +59129,470 @@
...
+        "minStat": 4,
+        "maxStat": 7
+      },
+      {
+        "stat": "Critical Damage",
+        "minStat": 9,
+        "maxStat": 12
+      },
+      {
+        "stat": "% Air Resistance",
```
Comment (samlee405) [inline server/app/database/data/weapons.json:59444]: % Neutral Resistance

File: server/app/database/data/weapons.json [hunk 14]
```diff
@@ -59100,5 +59129,470 @@
...
+      {
+        "stat": "% Fire Resistance",
+        "minStat": 5,
+        "maxStat": 7
+      }
+    ],
+    "weaponStats": {
+      "apCost": 5,
+      "usesPerTurn": 1,
+      "minRange": 1,
```
Comment (samlee405) [inline server/app/database/data/weapons.json:59457]: null

File: server/app/database/data/weapons.json [hunk 15]
```diff
@@ -59100,5 +59129,470 @@
...
+          "maxStat": 50
+        },
+        {
+          "stat": "Fire Steal",
+          "minStat": 4,
+          "maxStat": 5
+        },
+        {
+          "stat": "MP (target)",
+          "minStat": -1,
```
Comment (samlee405) [inline server/app/database/data/weapons.json:59474]: stat steal to just MP and min stat to null

File: server/app/database/data/weapons.json [hunk 16]
```diff
@@ -59100,5 +59129,470 @@
...
+        "minStat": 4,
+        "maxStat": 7
+      },
+      {
+        "stat": "Critical Damage",
+        "minStat": 9,
+        "maxStat": 12
+      },
+      {
+        "stat": "% Air Resistance",
```
Comment (samlee405) [inline server/app/database/data/weapons.json:59556]: % Neutral Resistance

File: server/app/database/data/weapons.json [hunk 17]
```diff
@@ -59100,5 +59129,470 @@
...
+          "minStat": 38,
+          "maxStat": 45
+        },
+        {
+          "stat": "Water Steal",
+          "minStat": 4,
+          "maxStat": 5
+        },
+        {
+          "stat": "AP (target)",
```
Comment (samlee405) [inline server/app/database/data/weapons.json:59585]: AP

File: server/app/database/data/items.json [hunk 18]
```diff
@@ -132514,5 +132639,1109 @@
...
+        "maxStat": 8
+      },
+      {
+        "stat": "Lock",
+        "minStat": 2,
+        "maxStat": 3
+      },
+      {
+        "stat": "Dodge",
+        "minStat": -6,
```
Comment (samlee405) [inline server/app/database/data/items.json:132713]: null

File: server/app/database/data/items.json [hunk 19]
```diff
@@ -132514,5 +132639,1109 @@
...
+        "maxStat": -6
+      },
+      {
+        "stat": "AP Parry",
+        "minStat": -4,
```
Comment (samlee405) [inline server/app/database/data/items.json:132718]: null

File: server/app/database/data/items.json [hunk 20]
```diff
@@ -132514,5 +132639,1109 @@
...
+        "maxStat": 2
+      },
+      {
+        "stat": "Critical Resistance",
+        "minStat": 9,
+        "maxStat": 12
+      },
+      {
+        "stat": "Pushback Resistance",
+        "minStat": -24,
```
Comment (samlee405) [inline server/app/database/data/items.json:132733]: null

File: server/app/database/data/items.json [hunk 21]
```diff
@@ -132514,5 +132639,1109 @@
...
+        "maxStat": 4
+      },
+      {
+        "stat": "% Fire Resistance",
+        "minStat": 3,
+        "maxStat": 4
+      },
+      {
+        "stat": "% Water Resistance",
+        "minStat": -2,
```
Comment (samlee405) [inline server/app/database/data/items.json:132748]: null

File: server/app/database/data/items.json [hunk 22]
```diff
@@ -132514,5 +132639,1109 @@
...
+        "maxStat": -2
+      },
+      {
+        "stat": "% Air Resistance",
+        "minStat": -2,
```
Comment (samlee405) [inline server/app/database/data/items.json:132753]: null

File: server/app/database/data/items.json [hunk 23]
```diff
@@ -132514,5 +132639,1109 @@
...
+        "maxStat": 3
+      },
+      {
+        "stat": "AP Parry",
+        "minStat": 1,
+        "maxStat": 2
+      },
+      {
+        "stat": "MP Parry",
+        "minStat": -4,
```
Comment (samlee405) [inline server/app/database/data/items.json:132845]: null

File: server/app/database/data/items.json [hunk 24]
```diff
@@ -132514,5 +132639,1109 @@
...
+        "maxStat": -4
+      },
+      {
+        "stat": "Critical Resistance",
+        "minStat": -24,
```
Comment (samlee405) [inline server/app/database/data/items.json:132850]: null

File: server/app/database/data/items.json [hunk 25]
```diff
@@ -132514,5 +132639,1109 @@
...
+        "maxStat": 12
+      },
+      {
+        "stat": "% Earth Resistance",
+        "minStat": -2,
+        "maxStat": -2
+      },
+      {
+        "stat": "% Fire Resistance",
+        "minStat": -2,
```
Comment (samlee405) [inline server/app/database/data/items.json:132865]: null for the min

File: server/app/database/data/items.json [hunk 26]
```diff
@@ -132514,5 +132639,1109 @@
...
+        "maxStat": 4
+      },
+      {
+        "stat": "Dodge",
+        "minStat": -8,
+        "maxStat": -8
+      },
+      {
+        "stat": "AP Parry",
+        "minStat": -8,
```
Comment (samlee405) [inline server/app/database/data/items.json:132967]: null for min

File: server/app/database/data/items.json [hunk 27]
```diff
@@ -132514,5 +132639,1109 @@
...
+        "maxStat": 4
+      },
+      {
+        "stat": "Critical Resistance",
+        "minStat": 7,
+        "maxStat": 10
+      },
+      {
+        "stat": "Pushback Resistance",
+        "minStat": -20,
```
Comment (samlee405) [inline server/app/database/data/items.json:132982]: null

File: server/app/database/data/items.json [hunk 28]
```diff
@@ -132514,5 +132639,1109 @@
...
+        "maxStat": 8
+      },
+      {
+        "stat": "% Water Resistance",
+        "minStat": -4,
+        "maxStat": -4
+      },
+      {
+        "stat": "% Air Resistance",
+        "minStat": -4,
```
Comment (samlee405) [inline server/app/database/data/items.json:133002]: null for these two lines as well

File: server/app/database/data/items.json [hunk 29]
```diff
@@ -132514,5 +132639,1109 @@
...
+        "maxStat": 6
+      },
+      {
+        "stat": "Air Damage",
+        "minStat": 4,
+        "maxStat": 6
+      },
+      {
+        "stat": "Lock",
+        "minStat": -8,
```
Comment (samlee405) [inline server/app/database/data/items.json:133084]: null

File: server/app/database/data/items.json [hunk 30]
```diff
@@ -132514,5 +132639,1109 @@
...
+        "maxStat": 4
+      },
+      {
+        "stat": "MP Parry",
+        "minStat": -8,
+        "maxStat": -8
+      },
+      {
+        "stat": "Critical Resistance",
+        "minStat": -20,
```
Comment (samlee405) [inline server/app/database/data/items.json:133104]: here s well

File: server/app/database/data/items.json [hunk 31]
```diff
@@ -132514,5 +132639,1109 @@
...
+        "maxStat": 10
+      },
+      {
+        "stat": "% Earth Resistance",
+        "minStat": -4,
+        "maxStat": -4
+      },
+      {
+        "stat": "% Fire Resistance",
+        "minStat": -4,
```
Comment (samlee405) [inline server/app/database/data/items.json:133119]: these two lines as well

File: server/app/database/data/items.json [hunk 32]
```diff
@@ -132514,5 +132639,1109 @@
...
+        "maxStat": 10
+      },
+      {
+        "stat": "Air Damage",
+        "minStat": 7,
+        "maxStat": 10
+      },
+      {
+        "stat": "Lock",
+        "minStat": -10,
```
Comment (samlee405) [inline server/app/database/data/items.json:133206]: null

File: server/app/database/data/items.json [hunk 33]
```diff
@@ -132514,5 +132639,1109 @@
...
+        "maxStat": 3
+      },
+      {
+        "stat": "MP Parry",
+        "minStat": -6,
+        "maxStat": -6
+      },
+      {
+        "stat": "Critical Resistance",
+        "minStat": -16,
```
Comment (samlee405) [inline server/app/database/data/items.json:133226]: again here

File: server/app/database/data/items.json [hunk 34]
```diff
@@ -132514,5 +132639,1109 @@
...
+        "maxStat": 8
+      },
+      {
+        "stat": "% Earth Resistance",
+        "minStat": -3,
+        "maxStat": -3
+      },
+      {
+        "stat": "% Fire Resistance",
+        "minStat": -3,
```
Comment (samlee405) [inline server/app/database/data/items.json:133241]: again here

File: server/app/database/data/items.json [hunk 35]
```diff
@@ -132514,5 +132639,1109 @@
...
+        "maxStat": 5
+      },
+      {
+        "stat": "Dodge",
+        "minStat": -10,
+        "maxStat": -10
+      },
+      {
+        "stat": "AP Parry",
+        "minStat": -6,
```
Comment (samlee405) [inline server/app/database/data/items.json:133338]: here on these 2

File: server/app/database/data/items.json [hunk 36]
```diff
@@ -132514,5 +132639,1109 @@
...
+        "maxStat": 3
+      },
+      {
+        "stat": "Critical Resistance",
+        "minStat": 5,
+        "maxStat": 8
+      },
+      {
+        "stat": "Pushback Resistance",
+        "minStat": -16,
```
Comment (samlee405) [inline server/app/database/data/items.json:133353]: null

File: server/app/database/data/items.json [hunk 37]
```diff
@@ -132514,5 +132639,1109 @@
...
+        "maxStat": 6
+      },
+      {
+        "stat": "% Fire Resistance",
+        "minStat": 4,
+        "maxStat": 6
+      },
+      {
+        "stat": "% Water Resistance",
+        "minStat": -3,
```
Comment (samlee405) [inline server/app/database/data/items.json:133368]: null

File: server/app/database/data/items.json [hunk 38]
```diff
@@ -132514,5 +132639,1109 @@
...
+        "maxStat": -3
+      },
+      {
+        "stat": "% Air Resistance",
+        "minStat": -3,
```
Comment (samlee405) [inline server/app/database/data/items.json:133373]: null

File: server/app/database/data/items.json [hunk 39]
```diff
@@ -132514,5 +132639,1109 @@
...
+      "it": "[!] Dofus du Cauchemar",
+      "pt": "[!] Dofus du Cauchemar"
+    },
+    "itemType": "Dofus",
+    "setID": null,
+    "level": 180,
+    "stats": [
+      {
+        "stat": "Initiative",
+        "minStat": 1000,
```
Comment (samlee405) [inline server/app/database/data/items.json:133400]: null

File: server/app/database/data/items.json [hunk 40]
```diff
@@ -132514,5 +132639,1109 @@
...
+        "maxStat": 6
+      },
+      {
+        "stat": "AP Parry",
+        "minStat": 4,
+        "maxStat": 5
+      },
+      {
+        "stat": "Critical Resistance",
+        "minStat": -8,
```
Comment (samlee405) [inline server/app/database/data/items.json:133654]: -11 is min, -8 is max

File: server/app/database/data/items.json [hunk 41]
```diff
@@ -132514,5 +132639,1109 @@
...
+        "maxStat": 5
+      },
+      {
+        "stat": "Initiative",
+        "minStat": 151,
+        "maxStat": 200
+      },
+      {
+        "stat": "Critical Resistance",
+        "minStat": -8,
```
Comment (samlee405) [inline server/app/database/data/items.json:133731]: reverse min and max

**Review by samlee405 — CHANGES_REQUESTED** [review]
noted every change i could find.

File: server/app/database/data/items.json (hunk 40 above)
Comment (Josemanki) [reply server/app/database/data/items.json:133654]: I'm guessing this is because -8 is the one closest to 0, therefore we count it as a bigger stat?

Comment (samlee405) [reply server/app/database/data/items.json:133654]: Correct

**Review by samlee405 — APPROVED** [review]

**Pull request merged**